australian-guide-to-legal-citation.csl

```diff
@@ -199,11 +196,10 @@
         <text variable="title" font-style="italic" text-case="title" form="short"/>
       </if>
       <else>
-        <text variable="title" quotes="true" text-case="title" form="short"/>
+        <text variable="title" form="short" text-case="title" quotes="false"/>
```

Comment:
This is definitely how the style does it: http://www.law.unimelb.edu.au/files/dmfile/FinalOnlinePDF-2012Reprint.pdf#page=73 (rule 2.1.14)

You're absolutely right on the comments though. I'll fix that up when I get to work. Sorry. That was a dumb oversight.

Tim